docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
We run customer environments today where we definitely do have image on
separate partition.  For the few environments I checked, we were still
running with the default image gc values.  I need to speak to a couple
other operators on Monday that were unavailable today to ensure a single
value worked.  At worse, if we have to separate its just a separate signal
that would probably only have meaning when they are not unified.  Either
way, I want to close on this Monday.

On Friday, May 13, 2016, Dawn Chen <notifications@github.com> wrote:

> In docs/proposals/kubelet-eviction.md
> <https://github.com/kubernetes/kubernetes/pull/25237#discussion_r63263245>
> :
>
> > @@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
> >  | Eviction Signal  | Description                                                                     |
> >  |------------------|---------------------------------------------------------------------------------|
> >  | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
> > +| disk.available   | disk.available := node.stats.fs.available |
>
> Discussed this with @vishh <https://github.com/vishh>: my concern is that
> one number is hard to choose if the node is configured to have different
> partitions for images. If the number is too small, we end up to evict an
> image to get out of the out-of-disk situation, and pull another image to
> back to the bad state again. But I agreed with Vish today, the root fs is
> used for emptydir too, which could be big too. We can iterate this later.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25237/files/e7053d1db93e2b92f27a777bc230a6a8ed901cba#r63263245>
>